modules/asio/1.32.0/overlay/BUILD.bazel

```diff
@@ -0,0 +1,17 @@
+load("@rules_cc//cc:defs.bzl", "cc_library")
```

Comment:
Did this file change? If not, could you replace it with a symlink to the previous version?